setup.cfg

```
@@ -14,27 +14,28 @@ classifiers =
     Development Status :: 4 - Beta
     Framework :: Django
     Framework :: Django :: 3.2
-    Framework :: Django :: 4.0
+    Framework :: Django :: 4.1
+    Framework :: Django :: 4.2
     Intended Audience :: Developers
     Operating System :: Unix
     Operating System :: MacOS
     Operating System :: Microsoft :: Windows
-    Programming Language :: Python :: 3.7
     Programming Language :: Python :: 3.8
     Programming Language :: Python :: 3.9
     Programming Language :: Python :: 3.10
+    Programming Language :: Python :: 3.11
     Topic :: Software Development :: Libraries :: Python Modules
 
 [options]
 zip_safe = False
 include_package_data = True
 packages = find:
 install_requires =
-    django>=3.1.0
+    Django >=3.2, <4.3
```

django follows a deterministic versioning scheme - `X.0, X.1, X.2`, so there will never be a `4.3`. You would rather write this as `< 5.0`, BUT, I don't see the reason to pin the upper version of Django? It could be that Django 5.0 will just work out of the box and that's a responsibility for the project using this dependency to manage.

```suggestion
    Django >=3.2
```